### PR TITLE
Structure improvement, simpler language.

### PR DESCRIPTION
**Pull request by rkratky**
### What does this PR do?

* Improves structuring
* Simplifies language

### What issues does this PR fix or reference?

Modifies https://github.com/eclipse/che-docs/pull/899

### CODE REVIEW

File: src/main/pages/che-7/installation-guide/proc_configuring-namespace-strategies.adoc [hunk 1]
```diff
@@ -41,6 +41,8 @@ CHE_INFRA_KUBERNETES_NAMESPACE_DEFAULT=che-workspaces
 ----
 ====
 
+IMPORTANT: To run more than one workspace at a time when using this strategy together with the `common` PVC strategy, configure persistent volumes to use `ReadWriteMany` access mode.
```
Comment (metlos) [inline src/main/pages/che-7/installation-guide/proc_configuring-namespace-strategies.adoc:44]: I'd add something along the lines of `Alternatively, limit the number of concurrently running workspaces per user to 1 when using _One namespace per user_ namespace strategy.`

The reason is that `ReadWriteMany` is not even available on some k8s installations or kinds of persistent volumes, so we should offer a solution that works around that limitation.

File: src/main/pages/che-7/installation-guide/proc_configuring-namespace-strategies.adoc [hunk 2]
```diff
@@ -59,14 +61,7 @@ CHE_INFRA_KUBERNETES_NAMESPACE_DEFAULT=che-ws-__<username>__
...
-concurrently. Because this might be tricky to set up reliably you may want to use a different PVC strategy or, in case
-of _One namespace per user_, to limit the number of concurrently running workspaces per user to 1.
-
-To enforce this limit, set:
+To run more than one workspace at a time when using this strategy together with the `common` PVC strategy, configure persistent volumes to use `ReadWriteMany` access mode.
 
-----
-CHE_LIMITS_USER_WORKSPACES_RUN_COUNT=1
-----
+To limit the number of concurrently running workspaces per user to one, set the `CHE_LIMITS_USER_WORKSPACES_RUN_COUNT` environment variable to `1`.
```
Comment (metlos) [inline src/main/pages/che-7/installation-guide/proc_configuring-namespace-strategies.adoc:66]: I'm not sure if `CHE_LIMITS_USER_WORKSPACE_RUN_COUNT` is mentioned anywhere else, that's why I kept it here.

Comment (sleshchenko) [reply src/main/pages/che-7/installation-guide/proc_configuring-namespace-strategies.adoc:66]: I'm not sure that we should mention env var here at all. It seems to be to low level...
I mean the user should understand where to find the right place to investigate how to deliver such an environment variable according to deployment method -> che-operator, helm chart.
With helm chart - there is no ability to configure this env var, eventually, we should add some configurable value into helm chart.
For che operator - it's customCheProperties field in CheCluster CR https://github.com/eclipse/che-docs/blob/569dec8754b86c28b9b93e82b1f16e0e516b37f4/src/main/pages/che-7/installation-guide/con_che-configmaps-and-their-behavior.adoc#che-installed-using-an-operator

So, I would say here we should have something like
```
To limit the number of concurrently running workspaces per user to one configure:
  - `.global.workspace.number` for helm chart. .... reference to docs that describes how to set value for helm chart.
  - `spec.server.cheCustomProperties.CHE_LIMITS_USER_WORKSPACE_RUN_COUNT` for che operator .... reference to docs that describes how to set value for CheCluster CR
```

File: src/main/pages/che-7/installation-guide/proc_configuring-namespace-strategies.adoc (hunk 1 above)
Comment (rkratky) [reply src/main/pages/che-7/installation-guide/proc_configuring-namespace-strategies.adoc:44]: > I'd add something along the lines of Alternatively, limit the number of concurrently running workspaces per user to 1

@metlos, I believe this is sufficiently implied by *To run more than one workspace at a time*.

File: src/main/pages/che-7/installation-guide/proc_configuring-namespace-strategies.adoc (hunk 2 above)
Comment (rkratky) [reply src/main/pages/che-7/installation-guide/proc_configuring-namespace-strategies.adoc:66]: @metlos, do you agree with @sleshchenko's suggestion? Then we could perhaps merge.

Comment (metlos) [reply src/main/pages/che-7/installation-guide/proc_configuring-namespace-strategies.adoc:66]: :+1:

Comment (metlos) [reply src/main/pages/che-7/installation-guide/proc_configuring-namespace-strategies.adoc:66]: I have no idea about the locations of the mentioned docs though...

**Comment by rkratky** [comment]
@metlos, I applied @sleshchenko's suggestion (links to respective Helm and Operator docs to be added later). Please, merge this PR.

**Comment by rkratky** [comment]
@metlos, I hate to nag, but is there anything I should modify in this PR before it can be merged?

File: src/main/pages/che-7/installation-guide/proc_configuring-namespace-strategies.adoc [hunk 3]
```diff
@@ -59,14 +61,14 @@ CHE_INFRA_KUBERNETES_NAMESPACE_DEFAULT=che-ws-__<username>__
...
 
-----
-CHE_LIMITS_USER_WORKSPACES_RUN_COUNT=1
-----
+// Links to Helm and Operator docs need to be added below:
+
+To limit the number of concurrently running workspaces per user to one (1):
+
+* For Helm Chart deployments: set the `.global.workspace.number` parameter to `1`.
+* For Operator deployments: set the `spec.server.cheCustomProperties.CHE_LIMITS_USER_WORKSPACE_RUN_COUNT` variable of the CheCluster Custom Resource (RC) to `1`.
```
Comment (metlos) [inline src/main/pages/che-7/installation-guide/proc_configuring-namespace-strategies.adoc]: Is `RC` the shortcut we're using in the docs for a custom resource? I'm usually seeing `CR` in the wild.

Comment (rkratky) [reply src/main/pages/che-7/installation-guide/proc_configuring-namespace-strategies.adoc]: No, that's just my typo :/ Fixing now.

**Pull request merged**